Resources/Private/JavaScript/asset-upload/src/components/FilePreview.module.css

```diff
@@ -1,25 +1,20 @@
-.fileList {
-    margin-top: var(--theme-spacing-Full);
+.preview {
     display: flex;
     flex-direction: row;
-    flex-wrap: wrap;
-}
-
-.fileListHeader {
-    flex: 1 1 100%;
-    margin-bottom: var(--theme-spacing-Full);
-    font-size: var(--theme-fontSize-base);
+    column-gap: 1rem;
+    padding-bottom: 1rem;
+    border-bottom: 1px solid #3f3f3f;
 }
 
 .thumb {
     display: inline-flex;
     border-radius: 2px;
     border: 1px solid #eaeaea;
-    margin-bottom: var(--theme-spacing-Half);
-    margin-right: var(--theme-spacing-Half);
-    width: 100px;
-    height: 100px;
-    padding: var(--theme-spacing-Quarter);
+    margin-bottom: var(--theme-spacing-Half, 8px);
+    margin-right: var(--theme-spacing-Half, 8px);
+    width: 270px;
```

Comment:
This should only be larger if the new upload is desired. So we need an additional class on the thumb in the preview.

Reply:
I added an additional class. This class is added when the property field should be shown in the upload dialog and the thumb should be wider.